integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/metamodel/IndexDescriptorIT.java

```diff
@@ -68,11 +71,28 @@ public void staticFields() {
 		Optional<IndexFieldDescriptor> objectFieldDescriptorOptional = indexDescriptor.field( "myObjectField" );
 		assertThat( objectFieldDescriptorOptional ).isPresent();
 
+		List<IndexFieldDescriptor> descriptors = new ArrayList<>();
+
+		if ( TckConfiguration.get().getBackendFeatures().supportsImplicitFields() ) {
+			Optional<IndexFieldDescriptor> idDescriptorOptional = indexDescriptor.field( "_id" );
+			assertThat( idDescriptorOptional ).isPresent();
+
+			Optional<IndexFieldDescriptor> indexDescriptorOptional = indexDescriptor.field( "_index" );
+			assertThat( indexDescriptorOptional ).isPresent();
+
+			Optional<IndexFieldDescriptor> entityTypeDescriptorOptional = indexDescriptor.field( "_entity_type" );
+			assertThat( entityTypeDescriptorOptional ).isPresent();
+
+			descriptors.add( idDescriptorOptional.get() );
+			descriptors.add( indexDescriptorOptional.get() );
+			descriptors.add( entityTypeDescriptorOptional.get() );
+		}
+
+		descriptors.add( valueFieldDescriptorOptional.get() );
+		descriptors.add( objectFieldDescriptorOptional.get() );
```

Comment:
I would move this to an Elasticsearch-specific test, since this is obviously specific to Elaticsearch: even if another, hypothetical backend supported the "implicit fields" feature, the fields would be different.

As to the assertion below, you can simply relax it: use `contains` instead of `containsExactlyInAnyOrder`?